Data/Dto/PostDto.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+
+namespace Blogger.Data.Dto
+{
+    public class PostDto : IRequestDto<Post>
+    {
+
+        public string Title {set; get;}
+        public string Body {set; get;}
+        public DateTime CreatedOn {set; get;}
+        
+        public dynamic Errors = null;
+
+        public bool IsValid()
+        {
+            if(IsValidTitle() && IsValidBody())
```

Comment:
It's a good practice to have a single return point from a method.

var isValid = IsValidTitle() && IsValidBody();
return isValid;

or 
return IsValidTitle() && IsValidBody();